Winget-AutoUpdate/winget-upgrade.ps1

```diff
@@ -139,6 +162,24 @@ if (Test-Network) {
     }
 }
 
+#Run WAU in user context if currently as system
+if ($IsSystem) {
+
+    #Get Winget system apps to excape them befor running user context
+    Get-WingetSystemApps
+
+    #Run user context scheduled task
+    $UserScheduledTask = Get-ScheduledTask -TaskName "Winget-AutoUpdate-UserContext" -ErrorAction SilentlyContinue
+    if ($UserScheduledTask){
+        Write-Log "Starting WAU in User context"
+        Start-ScheduledTask $UserScheduledTask.TaskName -ErrorAction SilentlyContinue
+        Exit 0
+    }
+    else {
+        Write-Log "User context execution not installed"
+    }
+}
+
```

Comment:
I think here we should remove the `winget_system_apps.txt` file after user context run. Because if someone uses WAU at home as user with admin right, you can run WAU with your user account and manage all in one. You don't really need the system part (this is mainly for companies where users need system execution to install udates). What do you think @KnifMelti ?

Reply:
In fact, it needs to be determined during install script. If WAU is installed by system (via Intune, SCCM, GPO,...) then, we need to have WAU as it is now. But if WAU is installed by an admin user, then it could run all in one. Sort of optimization ^^
I will think about that for the v1.16.0

Reply:
Then make it a choice...
...because a lot of the home admins also have families with no admin rights and they're using the same family computer!

Reply:
ok, I had other possibilities in mind (to run different way regarding system, admin user and standard user), but we'll leave it like that for now 😅

Reply:
Ah, interesting!